packages/ibm-products/src/components/Datagrid/useActionsColumn.js

```diff
@@ -107,6 +108,7 @@ const useActionsColumn = (hooks) => {
                   {!isFetching && rowActions.length > 2 && (
                     <div>
                       <OverflowMenu
+                        align="top-right"
```

Comment:
The `rowAction` `align` option only applies when rendering individual buttons (around line 85). 

There is no feature currently available to modify the OverflowMenu's trigger button ( [•••] ) itself. This is a temporary, hard-coded fix to duplicate the feature above.